script/Ciel/Day 5/05_01_CIEL05_1B.txt

```diff
@@ -352,7 +352,7 @@ My mind freezes.
 -- Page 81, Offset 11083.
 -- 集まってきた彼等の後ろに、一息遅れてやってくるモノたちを見て、どちらが“生きている”のかを理解する。
 // 生きている tweaked a bit to reflect on 11076 better
-When I look behind the group of people and see the %{i}things%{/i} that are hot on their tail,
+When I look behind the group of people and see the %{i}things%{/i} that are hot on their tails,
```

Comment:
I think tail stays singular here, since it's a more metaphorical shared 'tail'.

Reply:
I just followed the example on mw for this one. Which do you prefer?
![image](https://github.com/Tsukihimates/Tsukihime-Translation/assets/90473809/5da6290d-d6b5-4ab1-a9c1-c41bb1e4a6aa)

Reply:
Eh it seems both are used. I guess plural is fine.